docs/user_guide/mitigation.rst

```diff
@@ -74,6 +74,109 @@ Preprocessing
    
 .. currentmodule:: fairlearn.preprocessing
 
+Preprocessing algorithms transform the dataset to mitigate possible unfairness
+present in the data.
+Preprocessing algorithms in Fairlearn follow the :class:`sklearn.base.TransformerMixin`
+class, meaning that they can :code:`fit` to the dataset and :code:`transform` it
+(or :code:`fit_transform` to fit and transform in one go).
+
+.. _correlation_remover:
+
+Correlation Remover
+~~~~~~~~~~~~~~~~~~~
+Sensitive features can be correlated with non-sensitive features in the dataset.
+By applying the :code:`CorrelationRemover`, these correlations are projected away
+whiledetails from the original data are retained as much as possible (as measured
+by the least-squares error). The user can control the level of projection via the
+:code:`alpha` parameter. In mathematical terms, assume we have the original dataset
+:math:`X` which contains a set of sensitive attributes :math:`S` and a set of
+non-sensitive attributes :math:`Z`. The removal of correlation is then
+mathematically described as:
```

Comment:
I would use either 'mathematical term' OR 'mathematically described'  but not both.